src/projection.ts

```diff
@@ -0,0 +1,45 @@
+
+import {VgProjectionType} from './vega.schema';
+
+export type ProjectionType = VgProjectionType;
+
+export interface Projection {
```

Comment:
Looks like  descriptions of some properties in the Vega docs are a bit more comprehensive and have links to related bl.ock.org examples